.github/workflows/windows.yml

```diff
@@ -124,9 +125,10 @@ jobs:
           # Should save cache only if run in the master branch of the base repo
           # github.ref_name is 'ref/PR_#' in case of the PR, and 'branch_name' when executed on push
           save: ${{ github.ref_name == 'master' && 'true' || 'false'  }}
-          key: ${{ github.job }}-windows
+          append-timestamp: true
+          key: ${{ github.job }}-${{ runner.os }}-common
           restore-keys: |
-            ${{ github.job }}-windows
+            ${{ github.job }}-${{ runner.os }}-common
 
       - name: CMake configure
```

Comment:
can we print scache stat somewhere?
looks like caches are not used now

Reply:
it is not used because we have not it on master yet (the 1st one will be saved when this PR is merged)